l10n_it_fatturapa_out/views/account_view.xml

```diff
@@ -7,7 +7,10 @@
         <field name="inherit_id" ref="account.view_invoice_tree" />
         <field name="arch" type="xml">
             <xpath expr="//field[@name='state']" position="before">
-                <field name="fatturapa_state" />
+                <field
+                    name="fatturapa_state"
```

Comment:
`is_sale_document()` probabilmente

Reply:
Puoi proporre la modifica anche solo qui?
Non mi viene in mente un modo per chiamare i metodi del model da dentro la vista, credo ci sia solo `context` e poco altro.